src/logger.cpp

```diff
@@ -30,6 +32,27 @@ inline void logger(const char* message, ...)
 	fflush(g_oFile);
 }
 
+std::string uint_to_hex(unsigned int i)
+{
+	std::stringstream stream;
+	stream << "0x"
+		<< std::setfill('0') << std::setw(sizeof(unsigned int) * 2)
+		<< std::hex << i;
+	return stream.str();
+}
+void logGameState(uintptr_t* time, uintptr_t* p1, uintptr_t* p2, uintptr_t* objectData[], ...)
```

Comment:
I don't think this needs the variadic arguments, if you are logging something specific (i'm assuming you just want to only log the time and player pointers)